lib/erb_lint/cli.rb

```diff
@@ -30,10 +30,25 @@ def initialize
     def run(args = ARGV)
       dupped_args = args.dup
       load_options(dupped_args)
+
+      if with_cache? && autocorrect?
+        failure!("cannot run autocorrect mode with cache")
+      end
+
       @files = @options[:stdin] || dupped_args
 
       load_config
 
+      @cache = Cache.new(@config, file_loader) if with_cache? || clear_cache?
+      if clear_cache?
+        if cache.cache_dir_exists?
+          cache.clear
+          success!("cache directory cleared")
+        else
+          failure!("cache directory already doesn't exist, skipped deletion.")
```

Comment:
This sentence doesn't sound like correct english? 

```suggestion
          failure!("cache directory doesn't exist, skipped deletion.")
```

Reply:
Yeah not sure how that ended up there, probably some copy pasta - fixed in the latest update.